code/python/main.py

```diff
@@ -0,0 +1,7 @@
+from bubblesort import bubble_sort
+
+array = [21, 50, 12, 31, 61, 40, 30]
+
+if __name__ == "__main__":
+    bubble_sort(array)
```

Comment:
Please print out the original array before printing the sorted array

Reply:
OK. I will make this change.